README.md

```diff
@@ -1,30 +1,92 @@
-Prep Linux
-===
+# [`rstp` module](https://app.viam.com/module/viam-labs/rtsp-cam)
+
+This module implements the [`"rdk:component:camera"` API](https://docs.viam.com/components/camera/) in an `rstp` model.
+With this model, you can configure a streaming camera with an h264 or h265 track for use with Viam.
+
+> [!NOTE]
+> For more information, see [Modular Resources](https://docs.viam.com/registry/#modular-resources).
+>
+> If your streaming camera uses the `x264` streaming video format, use the [`rtsp-h264` camera](https://app.viam.com/module/erh/viamrtsp) instead.
+
+## Requirements
```

Comment:
@seanavery Question that always confuses me with module repos: Are these requirements being executed as part the module setup for those configuring from registry (so I could move to "Local Setup" section), or are they a must-do for module usage even if you are configuring from registry?